build.gradle

```diff
@@ -15,47 +15,47 @@ tasks.withType(JavaCompile) {
 }
 
 idea {
-    module {
-        // Marks the already(!) added srcDir as "generated"
-        generatedSourceDirs += file('target/generated-sources/protobuf/java')
-        generatedSourceDirs += file('target/generated-sources/protobuf/grpc-java')
-    }
+	module {
+		// Marks the already(!) added srcDir as "generated"
+		generatedSourceDirs += file('target/generated-sources/main/java')
+		generatedSourceDirs += file('target/generated-sources/main/grpc-java')
+	}
 }
 
 buildscript {
-  repositories {
-    maven { url "http://repo.maven.apache.org/maven2" }
-  }
-  dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.7'
-  }
+	repositories {
+		maven { url "http://repo.maven.apache.org/maven2" }
+	}
+	dependencies {
+		classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.7'
+	}
 }
 
 sourceSets {
-    main {
-        proto {
-            srcDir {'proto'}
-            include {'**/*.proto'}
-        }
-        java {
-            // include self written and generated code
-            srcDirs 'src/main/java', 'target/generated-sources/protobuf/java' , 'target/generated-sources/protobuf/grpc-java'
-        }
-    }
+	main {
+		proto {
+			srcDir {'proto'}
+			include {'**/*.proto'}
+		}
+		java {
+			// include self written and generated code
+			srcDirs 'src/main/java', 'target/generated-sources/main/java' , 'target/generated-sources/main/grpc-java'
+		}
+	}
```

Comment:
I have to say that indentation changes are really annoying for code review

Reply:
yeah. The gradle file not correctly being formatted is also annoying.